ports/alicevision/portfile.cmake

```
@@ -0,0 +1,66 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO alicevision/AliceVision
+  REF v3.2.0
+  SHA512 448e20b2bb6623034ab89ea56b0c7fdf09af34d7d0d47740590fa9cd9aa2b6864f3950c4e968d260c660a41a2f5877c10edf8cfd0dcdd18e58ce289b7336e9ec
+  HEAD_REF master
+  PATCHES fix-dynamic-boost.patch
+)
+
+# MeshSDFilter
+vcpkg_from_github(
+  OUT_SOURCE_PATH MESHSDFILTER_SOURCE_PATH
+  REPO bldeng/MeshSDFilter
+  REF 71c3028eda6fc29517e658c54833e280f9abad51
+  SHA512  61367aab7117922b6f0f56d58e95cd328ac8177b184add11530b2ce9a8474be4947c7b27de135d7ad2980d6d49cc8a44b066eb673784dd553e1b9e78e994b3bb
+  HEAD_REF master
+)
+
+file(REMOVE_RECURSE "${SOURCE_PATH}/src/dependencies/MeshSDFilter")
+file(RENAME "${MESHSDFILTER_SOURCE_PATH}" "${SOURCE_PATH}/src/dependencies/MeshSDFilter")
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DALICEVISION_BUILD_SFM=OFF
```

```suggestion
        -DALICEVISION_BUILD_SFM=ON
```
This should be ON by default, it's the most important part of the software.